charts/nucliadb_writer/templates/writer.vs.yaml

```diff
@@ -10,22 +10,68 @@ spec:
   http:
     - name: nucliadb_writer
       match:
+        # KBs (only for onprem)
```

Comment:
I'm not sure if this is needed here.
Also, the POST `/kbs` and `/kbs/import` endpoint are missing (and has always been).

No action needed, just thoroughly going through all routes.